data/projects/documentation_extraction.md

```diff
@@ -21,25 +21,28 @@ errors and perform other aesthetical fixes.
 
 ##### PREPARATION/BONDING
 
-* Learn about how different languages have their own documentation standards.
-* Learn about the work already done in documentation extraction and parsing.
-* A concept of an efficient documentation parsing system is designed.
+* Investigate about docstring diff's cut-off bug.
+  Related: [#4187](https://github.com/coala/coala/issues/4187)
+* Fix `allow_missing_func_desc` in current implementation of DocStyleBear.
+* Investigate about existing DocStyleBear. (Check if DocStyleBear runs fine on coala and coala-bears repo)
+* Investigate about coala's project structure. (Write a blog introducing the same to the community)
 
 ##### CODING PHASE 1
 
-* DocumentationStyleBear should work for at least one language,
-  preferably python.
-* Malformed documentation should raise errors.
-  Related: [#2143](https://github.com/coala/coala/issues/2143)
+* Fix DocumentationExtraction related issues.
+  Related: [#18](https://gitlab.com/coala/GSoC-2017/issues/18)
+* Fix DocumentationComment related issues.
+  Related: [#19](https://gitlab.com/coala/GSoC-2017/issues/19)
 
 ##### CODING PHASE 2
 
-* A base class, that does the extraction and parsing, is implemented.
-  [#2659](https://github.com/coala/coala/issues/2659)
-
+* Rigorous testing of newly renovated DocumentationAPI.
+* A DocBaseClass(), which does extraction,parsing and diff generation, is implemented.
```

Comment:
* For better formatting remove the "()" and surround DocBaseClass with `
* Space after comma